doc/api/n-api.md

```diff
@@ -4060,7 +4060,8 @@ napi_status napi_queue_async_work(napi_env env,
 Returns `napi_ok` if the API succeeded.
 
 This API requests that the previously allocated work be scheduled
-for execution.
+for execution. Once it returns successfully, this API **must not be called again
```

Comment:
Micro-nit/question: Is "successfully" necessary here?

Reply:
I'm always a fan of leaving off the bold/italics/etc. Not blocking, but suggesting.

Reply:
"must not be called again" might stand some clarification? Like, will it throw? Or fail silently? Or result in a behavior that is undefined?

Reply:
Last nit: 

```suggestion
for execution. Once it returns successfully, do not call the API with
the same `_napi_async_work` item again.
```

Reply:
@Trott My understanding was that our style was to avoid addressing the reader. "do not call" is another way of saying "you", isn't it?